recipe/install_nvcc.sh

```diff
@@ -2,61 +2,98 @@
 
 set -xeuo pipefail
 
-# Default to using `nvcc` to specify `CUDA_HOME`.
-if [ -z ${CUDA_HOME+x} ]
-then
-    CUDA_HOME="$(dirname $(dirname $(which nvcc)))"
-fi
-
 # Set `CUDA_HOME` in an activation script.
 mkdir -p "${PREFIX}/etc/conda/activate.d"
-cat > "${PREFIX}/etc/conda/activate.d/${PKG_NAME}_activate.sh" <<EOF
+cat >"${PREFIX}/etc/conda/activate.d/${PKG_NAME}_activate.sh" <<EOF
```

Comment:
I don't think whitespace changes like this are needed, but I could be missing something. This happens on a few subsequent lines.

Reply:
I think it's just my code formatter doing this, will revert.